### PR TITLE
The relationship between the user.php and other models.

### PR DESCRIPTION
**Pull request by William-W-Gray**
Finished creating the relationship between the users.php and the other model.

### CODE REVIEW

**Comment by CourtneyFradreck** [comment]
LGTM!

**Review by michaeltsuro — COMMENTED** [review]
Can you revisit your relationships, a user has one address, may you just make sure they are all correct

**Comment by William-S-Gray** [comment]
> Can you revisit your relationships, a user has one address, may you just make sure they are all correct

ok sir

File: app/Models/User.php [hunk 1]
```diff
@@ -2,11 +2,19 @@
...
 
+use App\Models\Address;
+use App\Models\Cart;
+use App\Models\Order;
+use App\Models\OrderItem;
+use App\Models\Product;
+use App\Models\ProductCategory;
+use App\Models\ProductReview;
+use App\Models\Payment;
+
```
Comment (icep0ps) [inline app/Models/User.php:17]: Remove these since you are already in the models namespace

File: app/Models/User.php [hunk 2]
```diff
@@ -44,4 +52,48 @@ protected function casts(): array
...
     }
+
+    /**
+     * Define the relationships
+     */
+    
+    public function addresses()
+    {
+        return $this->hasMany(Address::class);
+    }
```
Comment (icep0ps) [inline app/Models/User.php:63]: User can only have one address

**Review by icep0ps — CHANGES_REQUESTED** [review]
Add the following

- return type to your methods
- PHPDoc block

**Pull request closed without merging**